[2022-03-18 04:46:05,246] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-10-21T04:00:00+00:00 [queued]>
[2022-03-18 04:46:05,301] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-10-21T04:00:00+00:00 [queued]>
[2022-03-18 04:46:05,304] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:05,305] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-03-18 04:46:05,306] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 04:46:05,356] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2014-10-21 04:00:00+00:00
[2022-03-18 04:46:05,367] {standard_task_runner.py:52} INFO - Started process 465 to run task
[2022-03-18 04:46:05,399] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2014-10-21T04:00:00+00:00', '--job-id', '6253', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmp91e4p5d_', '--error-file', '/tmp/tmpm5ujohur']
[2022-03-18 04:46:05,402] {standard_task_runner.py:77} INFO - Job 6253: Subtask get_pbp_data
[2022-03-18 04:46:05,573] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2014-10-21T04:00:00+00:00 [running]> on host a2ba0547c319
[2022-03-18 04:46:05,768] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2014-10-21T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2014-10-21T04:00:00+00:00
[2022-03-18 04:46:08,458] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 04:46:08,713] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20141021T040000, start_date=20220318T044605, end_date=20220318T044608
[2022-03-18 04:46:08,862] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 04:46:08,970] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
